modules/ossm-enabling-sidecar-injection.adoc

```diff
@@ -0,0 +1,261 @@
+// Module included in the following assemblies:
+// install/ossm-sidecar-injection
+
+:_mod-docs-content-type: PROCEDURE
+[id="ossm-enabling-sidecar-injection_{context}"]
+= Enabling sidecar injection
+
+The following procedures use the Bookinfo application to demonstrate different approaches for configuring sidecar injection.
```

Comment:
You should provide the "why" before the "what". So in this case, rewrite as: "To demonstrate different approaches for configuring sidecar injection, the following procedures use the Bookinfo application."